src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx

```diff
@@ -467,10 +467,14 @@ const ManageReputationDialogForm = ({
         <Button
           appearance={{ theme: 'primary', size: 'large' }}
           onClick={() => handleSubmit()}
-          text={{ id: 'button.confirm' }}
+          text={
+            values.forceAction || !isVotingExtensionEnabled
+              ? { id: 'button.confirm' }
+              : { id: 'button.createMotion' }
+          }
           loading={isSubmitting}
           disabled={!isValid || inputDisabled}
-          style={{ width: styles.wideButton }}
+          style={{ minWidth: styles.wideButton }}
```

Comment:
We usually prefer to add this via css styles not via JS.

You can do the same, by overwriting the button `min-width` in the styles file

Reply:
@rdig  So my goal was to remove the width style and replace it with minWidth. Are you saying that having the width style in the tsx file wasnt the best way to do it to begin with?

Reply:
Exactly. But now that I'm thinking more about this, in order to expedite this PR, leave it like it is for now.

Let's just fix your branch tommorow, then you can merge it